helix-term/src/ui/mod.rs

```diff
@@ -208,6 +210,34 @@ pub mod completers {
         names
     }
 
+    pub fn setting(input: &str) -> Vec<Completion> {
+        static KEYS: Lazy<Vec<Completion>> = Lazy::new(|| {
+            use serde_json::{Map, Value};
+            serde_json::de::from_slice::<Map<String, Value>>(
+                &serde_json::ser::to_vec(&Config::default()).unwrap(),
+            )
```

Comment:
```suggestion
            serde_json::from_slice::<Map<String, Value>>(
                &serde_json::to_vec(&Config::default()).unwrap(),
            )
```

Reply:
Also, why not `to_value` instead of `to_vec`, then `from_value` into a Map? Should be better since it's not serializing to string (which internally allocates these json::Values).

Reply:
Ah! Yes great suggestion I didn't knew this function existed :see_no_evil: 
https://github.com/helix-editor/helix/pull/798/commits/83405cf66a75181f1342de32cf99914bad309159